home/templates/home/index.html

```diff
@@ -1,44 +1,23 @@
-{% extends "base.html" %}
-{% load static %}
-{% block css %}
-<link rel="stylesheet" href="{% static 'css/home.css' %}" />
-{% endblock %}
-
-{% block content %}
-
-<div class="row">
-    <div class = "col-12 col-sm-12 col-md-12 col-lg-6">
-        <h1 class="mb-4">Upcoming Hackathons</h1>
-    </div>
-</div>
-
-<div class="row landing-page">
-
-    <div class="col-12 col-sm-12 col-md-12 col-lg-7">
-        {% if hackathons %}
-        {% for hackathon in hackathons %}
-            {% include 'hackathon/includes/hackathon_card.html' %}
-        {% endfor %}
-        <!-- Pagination using Django's paginator. Options are disabled or hidden if unavailable. -->
-        <nav aria-label="Hackathon Page Navigation">
-            {% include 'hackathon/includes/paginator.html' with page_obj=hackathons %}
-        </nav>
-        {% else %}
-        <div class="card">
-            <div class="card-body">
-                <div class="card-text">
-                    There aren't any Hackathons at the moment!
-                </div>
-            </div>
-        </div>
-        {% endif %}
-    </div>
-
-    <div class="col-12 col-sm-12 col-md-12 col-lg-5">
-        <div class="card">
-            <img class="card-img-top" src="{% static 'img/ci-hacking-new-normal--square.png' %}" alt="Code Institute Hackathon Logo">
-        </div>
-    </div>
-</div>
-
-{% endblock %}
+{% extends "base.html" %}
+{% load static %}
+{% block css %}
+<link rel="stylesheet" href="{% static 'css/home.css' %}" />
```

Comment:
Self-closing elements don't need the XHTML closing `/` at the end.
Also, consider adding `type="text/css"`

Reply:
Removed the `/` for self-closing tag.

Reply:
Removed the `/` for self-closing tag.